viewing_party/party.py

```diff
@@ -0,0 +1,117 @@
+def create_movie(movie_title, genre, rating):
+    new_movie = { }
+    if movie_title==None or genre==None or rating==None:
+        return None
+    new_movie['title'] = movie_title
+    new_movie['genre'] = genre
+    new_movie['rating'] = rating
+    return new_movie
+
+def add_to_watched(user_data, movie):
+    watched_list = user_data["watched"]
+    watched_list.append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    watch_list = user_data["watchlist"]
+    watch_list.append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    watch_list = user_data["watchlist"]
+    watched_list = user_data["watched"]
+    for movie_dict in watch_list:
+        if title==movie_dict["title"]:
+            watched_list.append(movie_dict)
+            watch_list.remove(movie_dict)
+    return user_data 
+
+def get_watched_avg_rating(user_data):
+    watched_list = user_data["watched"]
+    ratings = []
+    for movie_dict in watched_list:
+        ratings.append(movie_dict["rating"])
+    if ratings != []:
+        sum_of_ratings = sum(ratings)
+        num_of_ratings = len(ratings)
+        average = sum_of_ratings/num_of_ratings
+    else:
+        average = 0
+    return average
+
+def get_most_watched_genre(user_data):
+    genres = set()
```

Comment:
okay, I see you out here using sets! For readability, I would suggest putting an empty line between the for loops.